persist-cli/src/main/java/io/ballerina/persist/introspect/MySQLIntrospector.java

```diff
@@ -0,0 +1,158 @@
+/*
+ *  Copyright (c) 2024 WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.persist.introspect;
+
+
+import java.sql.Connection;
+
+public class MySQLIntrospector extends Introspector {
```

Comment:
Usually what we follow as the naming convention `MySqlIntrospector`

Please check other places as well.